release_config.yml

```diff
@@ -0,0 +1,13 @@
+community-tags:
+  - Virtual Schema
+  - Open Source
+  - GitHub
+community-project-name: Generic Virtual Schema
+community-project-description: |
+  The **Generic Virtual Schema** provides an abstraction layer that makes an external data source accessible from an Exasol database through regular SQL commands.
```

Comment:
Maybe you could add here that it is for data sources that offer an JDBC driver

Reply:
Nice idea, I'll add it